packages/edit-site/src/hooks/push-changes-to-global-styles/index.js

```diff
@@ -212,6 +214,19 @@ function PushChangesToGlobalStylesControl( {
 const withPushChangesToGlobalStyles = createHigherOrderComponent(
 	( BlockEdit ) => ( props ) => {
 		const blockEditingMode = useBlockEditingMode();
+
+		/**
+		 * Do not show the panel if the block does not have style support.
+		 *
+		 * @see https://github.com/WordPress/gutenberg/issues/52139
+		 */
+		if (
+			! hasBlockSupport( props.name, 'color' ) ||
```

Comment:
```suggestion
			! hasBlockSupport( props.name, 'color' ) &&
```

@Mamaduka  @richtabor What condition should we use here though?